api/internal/plugins/execplugin/execplugin_test.go

```diff
@@ -89,3 +92,32 @@ metadata:
 		t.Fatalf("unexpected arg array: %#v", p.Args())
 	}
 }
+
+func TestExecPlugin_ErrIfNotExecutable(t *testing.T) {
```

Comment:
@monopole Could you take a look. I think this test should work but not sure about the practice that skip it on Windows.

Reply:
yes, this is ok in this case.